src/AutoRest.CSharp/LowLevel/Output/LowLevelOutputLibraryFactory.cs

```diff
@@ -120,8 +121,8 @@ private static IReadOnlyList<ClientInfo> SetHierarchy(IReadOnlyDictionary<string
             {
                 var clientName = ClientBuilder.GetClientPrefix(context.DefaultLibraryName, context) + ClientBuilder.GetClientSuffix(context);
                 var clientNamespace = context.DefaultNamespace;
-                var endpointParameter = context.CodeModel.GlobalParameters.FirstOrDefault(RestClientBuilder.IsEndpointParameter);
-                var clientParameters = endpointParameter != null ? new[] { endpointParameter } : Array.Empty<RequestParameter>();
+                var endpointParameter = CodeModelConverter.CreateOperationParameters(context.CodeModel.GlobalParameters).FirstOrDefault(p => p.IsEndpoint);
```

Comment:
Hi, the input of LowLevelOutputLibraryFactory is the new codeModel (not m4 codeModel). Right? In this factory, all we operated in our new codeModel. and for swagger, we well convert m4 to new codeModel before the factory. Then in the factory, there is no need to convert anything. Is the design? (
for swagger:
 swagger->m4->new codeModel->lowlevelOutputLibray, 
for cadl, 
new codeModel->lowlevelOutputLibrary)

Reply:
@chunyu3 are you saying we are going to change the swagger behavior to accept new codeModel?

Reply:
Fixed. Now `LowLevelOutputLibraryFactory` uses `InputNamespace`: 
https://github.com/Azure/autorest.csharp/pull/2424/files#diff-7d0053c8a00c13755fb7fe14b79cb3794ac143885f2e48e3b126cfb60d9308c6R19